src/atoms/TextArea/TextArea.native.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextInput from '../TextInput';
+import automation from '../../_helpers/automation-attributes';
+
+const TextArea = ({
+  placeholder,
+  helpText,
+  errorText,
+  onChangeText,
+  variant,
+  prefix,
+  suffix,
+  disabled,
+  children,
+  iconLeft,
+  iconRight,
+  maxLength,
+  showCharacterCount,
+  label,
+  testID,
+  labelPosition,
+  size,
+}) => {
+  return (
+    <TextInput
```

Comment:
when `prefix` and `leftIcon` both are provided we need to throw an error same applies to `suffix` and `rightIcon`